bottlecap/src/config.rs

```diff
@@ -18,12 +18,12 @@ pub enum LogLevel {
     Error,
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
These types should probably both derive Copy as well, since they are plain old data types, (ie, don't own any heap memory/pointers).

Reply:
On it!